core/lib/generators/solidus/install/app_templates/payment_method/paypal.rb

```diff
@@ -3,7 +3,8 @@
 end
 
 unless Bundler.locked_gems.dependencies['solidus_paypal_commerce_platform']
-  bundle_command 'add solidus_paypal_commerce_platform'
+  version = @selected_frontend == 'classic' ? '< 1' : '>= 1.a'
```

Comment:
Why not simply `>= '1.0'`?

Reply:
Because we need to include the beta which would be excluded by `>= 1.0`:

```
>> Gem::Version.new('1.0.0.beta1') >= '1.0'
=> false
>> Gem::Version.new('1.0.0.beta1') >= '1.0.a'
=> true
>> 
```